x/wasm/internal/types/keys.go

```diff
@@ -60,3 +55,44 @@ func GetContractAddressKey(addr sdk.AccAddress) []byte {
 func GetContractStorePrefixKey(addr sdk.AccAddress) []byte {
 	return append(ContractStorePrefix, addr...)
 }
+
+// GetContractByCreatedSecondaryIndexKey returns the key for the secondary index:
+// `<prefix><codeID><created><contractAddr>`
+func GetContractByCreatedSecondaryIndexKey(contractAddr sdk.AccAddress, c *ContractInfo) []byte {
+	prefix := GetContractByCodeIDSecondaryIndexPrefix(c.CodeID)
```

Comment:
This is a bit tricky to follow because the term "prefix" does not match the one in the comment above. Seems correct but any naming improvement would be appreciated.

Reply:
This is a bit tricky indeed. This one generates a full key while the one above was the prefix used with the store.  To be more correct, I will rename the `GetContractStorePrefixKey` method to `GetContractStorePrefix` so that it is matching the other method signatures. Good 👁️

Reply:
Could we make the name in `<prefix><codeID><created><contractAddr>` more specific? Something like `<bucket_prefix>` or however you call those now? Then we free "prefix" as a term for any kind of prefix that changes depending on the context.